src/proof/proof_rule.h

```diff
@@ -232,6 +232,40 @@ enum class PfRule : uint32_t
    * above. \endverbatim
    */
   MACRO_SR_PRED_TRANSFORM,
+  /**
+   * \verbatim embed:rst:leading-asterisk
+   * **Builtin theory -- Encode predicate transformation**
+   * .. math::
+   *   \inferrule{F \mid G}{G}
+   * 
+   * where :math:`F` and :math:`G` are equivalent up to their encoding in an
+   * external proof format. This is currently verified by
+   *   :math:`\texttt{RewriteDbNodeConverter::convert}(F) = 
+   * \texttt{RewriteDbNodeConverter::convert}(G)`.
+   * This rule can be treated as a no-op when appropriate in external proof
+   * formats.
+   * \endverbatim
+   */
+  ENCODE_PRED_TRANSFORM,
+  /**
+   * \verbatim embed:rst:leading-asterisk
+   * **Builtin theory -- DSL rewrite**
+   * .. math::
+   *   \inferrule{F_1 \dots F_n \mid id t_1 \dots t_n}{F}
+   * 
```

Comment:
needs to be clang-formatted

Reply:
This is clang-formatted?

Reply:
Sorry, I had tested with `git clang-format main` and there are other parts of the file that are not formatted. But IIRC one has to be a bit careful how to break lines because of the sphinx stuff, so let's keep as is.